packages/ui/package.json

```diff
@@ -37,13 +37,14 @@
     "check:esm": "node --input-type=module --eval 'import \"@aws-amplify/ui\"'",
     "clean": "rimraf dist node_modules",
     "dev": "yarn build --watch",
-    "lint": "yarn typecheck",
+    "lint": "yarn typecheck && eslint src --ext .js,.ts,.tsx",
```

Comment:
I don't believe the UI package uses the internal linting config, meaning it would not be picking up the `import` rule updates. Can you confirm?

Reply:
Correct it's not using the internal linting config. I added the rule separately to the ui package's config.